src/aeroalpes/modulos/vuelos/dominio/reglas.py

```diff
@@ -24,23 +24,116 @@ def es_valido(self) -> bool:
                 return True
         return False
 
-class RutaValida(ReglaNegocio):
+class _RutaFechasValidas(ReglaNegocio):
+    ruta: Ruta
+
+    def __init__(self, ruta, mensaje='Las fechas de la ruta son incorrectas'):
+        super().__init__(mensaje)
+        self.ruta = ruta
+
+    def es_valido(self) -> bool:
+        return self.ruta.fecha_llegada() > self.ruta.fecha_salida()
 
+class _RutaLocalizacionesValidas(ReglaNegocio):
     ruta: Ruta
 
     def __init__(self, ruta, mensaje='La ruta propuesta es incorrecta'):
         super().__init__(mensaje)
         self.ruta = ruta
 
     def es_valido(self) -> bool:
-        return self.ruta.destino != self.ruta.origen
+        return not (self.ruta.origen() and self.ruta.destino()) or self.ruta.origen() != self.ruta.destino()
```

Comment:
Acá se verifica si es valido, no debería ser:
```suggestion
        return self.ruta.origen() and self.ruta.destino() and self.ruta.origen() != self.ruta.destino()
```